packages/__globalPreview/campaign.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr" class="su-page-campaign">
```

Comment:
Adding a preview template for the campaign page. Thanks for adding the class @iamrentman